docs/JSSystem.cpp.py

```diff
@@ -26,8 +26,8 @@
                                         ("sockets", "Open sockes", "integer"),
                                         ("files", "Open Files", "integer")]))
 )
-FunctionDoc( "System.language", "Get the system language.",
-    SeesDocs( "System.language" ),
+FunctionDoc( "NidiumSystem.language", "Get the system language.",
```

Comment:
No it's still System.
NidiumSystem is just the name of the class and should not be accessed directly.